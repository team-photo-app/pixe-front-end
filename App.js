import React from 'react';
import { Provider } from 'react-redux';
import store from './src/store/store';

import { AppLoading } from 'expo';
import * as Font from 'expo-font';
import { Ionicons } from '@expo/vector-icons';
import AppContainer from './src/NavRouter'


class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      isReady: false,
    };
  }

  async componentDidMount() {
    await Font.loadAsync({
      Roboto: require('native-base/Fonts/Roboto.ttf'),
      Roboto_medium: require('native-base/Fonts/Roboto_medium.ttf'),
      ...Ionicons.font,
    });
    this.setState({ isReady: true });
  }

  render() {
    return (
      this.state.isReady
        ? <Provider store={store}>
            <AppContainer />
            {/*<Gallery />*/}
            {/*<SignUp />*/}
          </Provider>
        : <AppLoading />
    );
  }
}


export default App;
