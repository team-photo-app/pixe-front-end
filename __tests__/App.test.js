/* eslint-disable no-undef */
import React from 'react'
import App from '../App.js'
import renderer from 'react-test-renderer'

it('App renders without crashing', () => {
  const rendered = renderer.create(<App />).toJSON()
  expect(rendered).toBeNull()
})

it('App test against snapshot', () => {
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
