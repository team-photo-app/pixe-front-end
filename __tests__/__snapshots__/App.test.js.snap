// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test against snapshot 1`] = `null`;
